service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseDataExtractionService.java

```diff
@@ -80,12 +79,14 @@ public CaseDataExtractionService(DateFormatterService dateFormatterService,
     public Map<String, Object> getStandardOrderDirectionData(CaseData caseData) throws IOException {
         ImmutableMap.Builder data = ImmutableMap.<String, Object>builder();
 
-        if (isNotEmpty(caseData.getStandardDirectionOrder())) {
-            data.put("judgeAndLegalAdvisor", prepareJudgeAndLegalAdvisor(
-                caseData.getStandardDirectionOrder().getJudgeAndLegalAdvisor()));
-        } else {
-            data.put("judgeAndLegalAdvisor", prepareJudgeAndLegalAdvisor(JudgeAndLegalAdvisor.builder().build()));
-        }
+        JudgeAndLegalAdvisor judgeAndLegalAdvisor = isNotEmpty(caseData.getStandardDirectionOrder())
```

Comment:
Why not to use `caseData.getStandardDirectionOrder() != null`? Using `isNotEmpty` makes me wonder how it can be empty. Makes me wonder whether it is a list or a map. Mind that as a ready I have no type information at this point.

Reply:
can do, I used existing implementation for reference so stuck with isNotEmpty. Will change to != null, I like that better too

Reply:
Someone made bad decision that was not verified :(